docs/README.md

```diff
@@ -0,0 +1,219 @@
+
+GoReleaser is built for CI tools, you only need a [single command line](#integration-with-ci) in your build script. Therefore, no package is required.  
+You can define your customization in a `goreleaser.yml` file. For examples, check the [goreleaser.example.yml](https://github.com/goreleaser/goreleaser/blob/master/goreleaser.example.yml) or the [goreleaser.yml](https://github.com/goreleaser/goreleaser/blob/master/goreleaser.yml) used by GoReleaser itself. More on this in [Release customization](#release-customization).
+We are also working on integrating package managers, we currently support Homebrew.
+
+The idea started with a simple shell script ([old GoReleaser](https://github.com/goreleaser/old-go-releaser)), but it quickly became more complex and I also wanted to publish binaries via Homebrew.
+
+_So, the all-new GoReleaser was born._
+
+##  Quick start
+
+In this example, we will build, archive and release a Golang project.  
+For simplicity, create a GitHub repository and add a single main package:
+```go
+// main.go
+package main
+
+func main() {
+  println("Ba dum, tss!")
+}
+```
+By default, GoReleaser will build the *main.go* file located in your current directory, but you can change the build package path in the GoReleaser configuration file.
+
+```yml
+# goreleaser.yml
+# Build customization
+build:
+  binary_name: drum-roll
+  goos:
+    - windows
+    - darwin
+    - linux
+  goarch:
+    - amd64
+```
+
+This configuration specifies the build operating systems to Windows, Linux and MacOS using 64bit architecture, the name of the binaries is "drum-roll".  
+
+GoReleaser will then archive the result binaries of each Os/Arch into a separate file. The default format is `{{.BinaryName}}_{{.Os}}_{{.Arch}}`.  
+You can change the archives name and format. You can also replace the OS and the Architecture with your own.  
+Another useful feature is to add files to archives, this is very useful for integrating assets like resource files.
+
+```yml
+# goreleaser.yml
+# Build customization
+build:
+  main: main.go
+  binary_name: drum-roll
+  goos:
+    - windows
+    - darwin
+    - linux
+  goarch:
+    - amd64
+# Archive customization
+archive:
+  format: tar.gz
+  replacements:
+    amd64: 64-bit
+    darwin: macOS
+    linux: Tux
+  files:
+    - /path/drum-roll.licence.txt
+```
+
+This configuration will generate tar archives, contains an additional file "drum-roll.licence.txt", the archives will be  located in:  
+  "./dist/drum-roll_windows_64-bit.tar.gz"  
+  "./dist/drum-roll_macOS_64-bit.tar.gz"  
+  "./dist/drum-roll_Tux_64-bit.tar.gz"  
+
+Next, you need to export a `GITHUB_TOKEN` environment variable with the `repo` scope selected. This will be used to deploy releases to your GitHub repository. Create yours [here](https://github.com/settings/tokens/new).  
```

Comment:
No comma after `Next`. Also consider: `next export a...` without the phrase `you need`- it scans better.